src/components/autocomplete/autocomplete.scss

```diff
@@ -51,14 +49,7 @@ md-autocomplete {
     height: auto;
 
     md-input-container {
-      padding-bottom: $input-container-padding + $input-error-height;
-
-      // When we have ng-messages, remove the input error height from our bottom padding, since the
-      // ng-messages wrapper has a min-height of 1 error (so we don't adjust height as often; see
-      // input.scss file)
-      &.md-input-has-messages {
-        padding-bottom: $input-container-padding;
-      }
+      padding-bottom: 0px;
```

Comment:
I think this should be `padding-bottom: $input-container-padding` so that it is bottom-aligned with the input. Based on the GIF you provided, it looks like the progress bar sits above the input's bottom border which is incorrect.